.husky/pre-commit

```diff
@@ -2,4 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 # npm test
-npm run auto-fix
+# npm run auto-fix
```

Comment:
Auto fix for some reason adds `"`'s, then `eslint` requires `'`. Can we remove the rule for now?

Reply:
my PR fixes this I added quote rules for prettier